### PR TITLE
Menu bar

### PR DESCRIPTION
**Pull request by aamir-asaram**
###Implemented Changes:
- [x] Made hamburger button clickable
- [x] Implemented fullscreen navbar as represented in Figma template
- [x] Linked navbar buttons to the relevant sections on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,8 +10,18 @@
...
 <body class="container">
   <header class="head">
+    <div id="menu" class="hide hide-desktop">
+      <img src="assets/images/Icon.png" class="close-btn">
+      <ul>
+        <li><a href="#works" class="top">Works</a></li>
+        <li><a href="#about" class="top">About</a></li>
+        <li><a href="#contact" class="top">Contact</a></li>
+      </ul>
+    </div>
```
Comment (LionRouge1) [inline index.html:20]: - [ ] Nice work implementing the mobile menu, it look really good :clap: :sunglasses: However the `color` and the `font-size` are different compared to the figma design. Kindly change the `color` to `#3A4A42` and the `font-size` to `32px` to follow the Figma design.
### Your design :point_down: 
Screenshot from 2023-05-16 09-36-10
### What it is supposed to look like :point_down: 
Screenshot from 2023-05-16 09-35-43

File: styles.css [hunk 2]
```diff
@@ -8,6 +8,60 @@
...
+
+li {
+  list-style: none;
+}
+
+#menu {
+  height: 100vh;
+  width: 100vw;
+  background-color: white;
+}
```
Comment (LionRouge1) [inline styles.css]:
```suggestion
#menu {
    height: 100vh;
    width: 100vw;
    background-color: white;
    position: fixed;
    overflow-y: hidden;
    top: 0;
    left: 0;
}
```
[OPTIONAL]
- [ ] You might consider making the mobile menu fixed and preventing it from scrolling with the page.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @aamir-asaram & @evansnyamekye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job implementing the mobile menu :+1: 
- Good commit :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @aamir-asaram, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Mobile menu works correctly 👍 
✅ Nice design🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**